apps/ejabberd/src/ejabberd_local_delivery.erl

```diff
@@ -0,0 +1,34 @@
+%%%-------------------------------------------------------------------
+%%% @author bartek
+%%% @copyright (C) 2016, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 21. Mar 2016 12:24
+%%%-------------------------------------------------------------------
+-module(ejabberd_local_delivery).
```

Comment:
I'm tempted to name this and other new modules here with `mongoose_` prefix and not `ejabberd_` as this is totally MongooseIM specific implementation. But I'd like to here from others @esl/mongooseim-developers